### PR TITLE
Fix various doxygen typos

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: src/dsp/sniffer_f.cpp [hunk 1]
```diff
@@ -101,7 +101,7 @@ int  sniffer_f::samples_available()
 /*! \brief Fetch available samples.
  *  \param out Pointer to allocated memory where the samples will be copied.
  *             Should be at least as big as buffer_size().
- *  \param num The number of sampels returned.
+ *  \param num The number of samplss returned.
```
Comment (argilo) [inline src/dsp/sniffer_f.cpp:104]:
```suggestion
 *  \param num The number of samples returned.
```

Comment (luzpaz) [reply src/dsp/sniffer_f.cpp]: Weird, the github UI has greyed out the button that merges this change. :confounded:

Comment (argilo) [reply src/dsp/sniffer_f.cpp]: That's because I merged it first. :smile:

**Review by argilo — APPROVED** [review]
Thanks for the fixes. I'll merge this once CI passes.

**Pull request merged**